services/responses.js

```diff
@@ -0,0 +1,341 @@
+      // If we're in this block, the error has a defined `status` property and
+      // the error should be directed out immediately
+      sendDefaultResponseForCode(err.status, err.message, res);
+    } else if (err.name === 'NotFoundError' ||
+      err.message.indexOf('ENOENT') !== -1 ||
+      err.message.indexOf('not found') !== -1) {
+      notFound(err, res);
+      // if the word "client" is ever in a message, it should be for the client.  We enforce that here.
+    } else if (err.message.indexOf('Client') !== -1) {
+      clientError(err, res);
+    } else {
+      serverError(err, res);
+    }
+  };
+}
+
+/**
+ * @param {number} code
+ * @param {string} message
+ * @param {object} res
+ * @param {object} extras
+ */
+function sendDefaultResponseForCode(code, message, res, extras) {
+  res.status(code).format({
+    json: sendJSONErrorCode(code, message, res, extras),
+    html: sendHTMLErrorCode(code, message, res),
+    text: sendTextErrorCode(code, message, res),
+    default: sendDefaultErrorCode(code, res)
+  });
+}
+
+/**
+ * Send whatever is default for this type of data with this status code.
+ * @param {number} code
+ * @param {object} res
+ * @returns {function}
+ */
+function sendDefaultErrorCode(code, res) {
+  return function () {
+    res.sendStatus(code);
+  };
+}
+
+/**
+ * Send some html (should probably be some default, or a render of a 500 page)
+ * @param {number} code
+ * @param {string} message
+ * @param {object} res
+ * @returns {function}
+ */
+function sendHTMLErrorCode(code, message, res) {
+  return function () {
+    res.type('html');
+    res.send(`${code} ${message}`);
+  };
+}
+
+/**
+ * Sends a default JSON message
+ * @param {number} code
+ * @param {string} message
+ * @param {object} res
+ * @param {object} extras
+ * @returns {function}
+ */
+function sendJSONErrorCode(code, message, res, extras) {
+  return function () {
+    res.json(_assign({ message, code }, extras));
+  };
+}
+
+/**
+ * Sends a default text message
+ * @param {number} code
+ * @param {string} message
+ * @param {object} res
+ * @returns {function}
+ */
+function sendTextErrorCode(code, message, res) {
+  return function () {
+    res.type('text');
+    res.send(code + ' ' + message);
+  };
+}
+
+/**
+ * All "Not Found" errors are routed like this.
+ * @param {Error} [err]
+ * @param {object} res
+ */
+function notFound(err, res) {
+  if (!(err instanceof Error) && err) {
+    res = err;
+  }
+
+  const message = 'Not Found',
+    code = 404;
+
+  // hide error from user of api.
+  sendDefaultResponseForCode(code, message, res);
+}
+
+/**
+ * All client errors should look like this.
+
+ * @param {Error} err
+ * @param {object} res
+ */
+function clientError(err, res) {
+  log('error', err.message, { stack: err.stack });
+
+  // They know it's a 400 already, we don't need to repeat the fact that its an error.
+  const message = removePrefix(err.message, ':'),
+    code = 400;
```

Comment:
maybe set this as a global variable at the top of the file, since you are using 400 in a couple of places like [here](https://github.com/clay/amphora-passport/pull/11/files#diff-c115819576e3d1e5ef8dd8fbeaa5384cR280) and [here](https://github.com/clay/amphora-passport/blob/770ddb026e6800227784f964eba8e20f7bf9769d/services/responses.js#L308)